app/src/main/java/toasty/messageinabottle/io/LiveBackend.java

```diff
@@ -54,4 +58,39 @@
             return result;
         }
     }
+
+    public static boolean tryToCreateAccount(String username, String password) throws IOException {
+        RequestBody formBody=new FormBody.Builder()
+                .add("username", username)
+                .add("password", password)
+                .build();
+        Request request=new Request.Builder()
+                .url("http://toastymmm.hopto.org/api/signup")
+                .post(formBody)
+                .build();
+        Response response=client.newCall(request).execute();
+        String result=response.body().string();
+
+        //if we get the response that it worked, then we created an account successfully,
+        //otherwise this username already exists.
+        return result.toLowerCase().contains("it worked");
```

Comment:
Instead of checking the body. We should be checking the response status code. i.e. 200 is okay